### ISSUE TITLE
Ecephys template should use BaseRecordingExtractorInterface

### ISSUE THREAD

**Issue by luiztauffer** [catalystneuro/to-nwb-template#4]
In most cases it's suggest that the user use a [`RecordingExtractorInterface`](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/nwb_conversion_tools/baserecordingextractorinterface.py) or [`SortingExtractorInterface`](https://github.com/catalystneuro/nwb-conversion-tools/blob/master/nwb_conversion_tools/basesortingextractorinterface.py) for electrophysiology data instead of building their own out of `BaseDataInterface`. 
The template should illustrate this

The same potentially for Ophys? @bendichter

**Comment by bendichter** [comment]
yeah, same thing. Using SegmentationExtractor or ImagingExtractor from [roiextractors](https://github.com/catalystneuro/roiextractors).
